packages/form-js-viewer/src/render/components/form-fields/Checklist.js

```diff
@@ -73,13 +73,7 @@ export function Checklist(props) {
           const isChecked = hasEqualValue(o.value, values);
 
           return (
-            <Label
-              htmlFor={itemDomId}
-              label={o.label}
-              class={classNames({
-                'fjs-checked': isChecked,
-              })}
-              required={false}>
+            <div className="fjs-inline-label" key={index}>
```

Comment:
Do not use indexes as keys, it can cause problems.
Let's use some variation of the `domId`